packages/cms/README.md

```diff
@@ -140,6 +140,14 @@ Keystone 底層是透過 [Prisma](https://github.com/prisma/prisma)來管理資
     docker start kids-cms;
     ```
 
+7. (optional) Remove the Docker container for the database migration.
+
+    ```bash
+    docker rm kids-cms-migration;
+    ```
+
+    you may check if the container is removed by running `docker ps -a`.
```

Comment:
Suggestion:
I think `docker stop kids-cms-migration` is optional as well.

Reply:
I added the `optional`s to the unnecessary steps in README